src/cascadia/TerminalSettingsEditor/ColorSchemes.cpp

```diff
@@ -233,75 +102,52 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
             {
```

Comment:
Why is this on the page? Shouldn't it be on the viewmodel?

Reply:
Removed!